test/integration/scheduler/scheduler_test.go

```diff
@@ -596,6 +596,62 @@ var _ = ginkgo.Describe("Scheduler", func() {
 		})
 	})
 
+	ginkgo.When("Creating objects out-of-order", func() {
```

Comment:
can we also have a test for out-or-order deletion of resources

Reply:
Are you asking to cover the finalizers? We have those in separate places.

Reply:
Yes, I was just thinking if we could have something analogous to this test; but maybe the ones we have are enough.